cmd/dmsg-server/commands/root.go

```diff
@@ -53,6 +59,22 @@ var rootCmd = &cobra.Command{
 		if err := sf.ParseConfig(os.Args, true, &conf); err != nil {
 			log.WithError(err).Fatal()
 		}
+		if sentryDSN != "" {
+			err := sentry.Init(sentry.ClientOptions{
+				Dsn: sentryDSN,
+				// Either set environment and release here or set the SENTRY_ENVIRONMENT
```

Comment:
What do you think about removing these comments and delete zero values from this structure?